tests/basic/src/main/res/menu/menu_main.xml

```diff
@@ -2,43 +2,6 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     tools:context=".MainActivity" >
-
```

Comment:
why was all the mirror cache things removed?

Reply:
We don't support MirrorCache.  It will have to be redone, if necessary, next year.